### PR TITLE
Updated firecracker-demo to v0.18.0 release

### PR DESCRIPTION
**Pull request by iulianbarbu**
#21 
Signed-off-by: iulianbarbu <iul@amazon.com>

Updated firecracker binary with the one compiled from tag v0.18.0.
Removed empty env variable (R_INIT) from start_firecracker.sh.
Added a note which clarifies the content added to /etc/security/limits.conf.

By submitting this pull request, I confirm that my contribution is made
under the terms of the Apache 2.0 license.

### CODE REVIEW

File: start-firecracker.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ RO_DRIVE="$PWD/xenial.rootfs.ext4"
 KERNEL="$PWD/vmlinux"
 TAP_DEV="fc-${SB_ID}-tap0"
 
-KERNEL_BOOT_ARGS="panic=1 pci=off reboot=k tsc=reliable quiet 8250.nr_uarts=0 ipv6.disable=1 $R_INIT"
-#KERNEL_BOOT_ARGS="console=ttyS0 reboot=k panic=1 pci=off nomodules ipv6.disable=1 $R_INIT"
+KERNEL_BOOT_ARGS="panic=1 pci=off reboot=k tsc=reliable quiet 8250.nr_uarts=0 ipv6.disable=1"
```
Comment (andreeaflorescu) [inline start-firecracker.sh]: Why is R_INIT deleted? Is this change related to updated to v0.18.0? If yes, can you please state that in your commit message?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ ec2-user hard nproc 16384
 EOL
 ```
 
+Note: In the above configuration, `ec2-user` it's a
```
Comment (andreeaflorescu) [inline README.md]: This change isn't related to updating Firecracker version and should be added in a separate commit.

Comment (iulianbarbu) [reply README.md]: New commit added with this specifc change.

File: start-firecracker.sh (hunk 1 above)
Comment (iulianbarbu) [reply start-firecracker.sh]: `Updated unused env variable from start_firecracker.sh.` from PR description was meant to describe this change.
I will update the description to `Removed empty env variable (R_INIT) from start_firecracker.sh.`. I will add this in the commit message also.

File: README.md (hunk 2 above)
Comment (dizz) [inline README.md]: english-wise you'd say " `ec2-user` **is** a placeholder..."

Comment (iulianbarbu) [reply README.md]: Sure. My mistake. I'll update.
